app/page.tsx

```diff
@@ -0,0 +1,77 @@
+import Image from "next/image";
+
+export default function Home() {
+  return (
+    <div className="flex min-h-screen flex-col items-center justify-center py-2">
```

Comment:
We can remove this default structure and have have just "Frequently Asked Questions" inside h1 tag

Reply:
Just migrating for now, this will come in a different PR.